homeassistant/components/cloud/__init__.py

```diff
@@ -294,22 +294,14 @@ async def async_startup_repairs(_: datetime) -> None:
     }
 
     async def _on_start() -> None:
-        """Discover platforms."""
+        """Handle cloud started after login."""
         nonlocal loaded
 
         # Prevent multiple discovery
         if loaded:
             return
         loaded = True
 
-        tts_info = {"platform_loaded": tts_platform_loaded}
```

Comment:
Is it still needed to have this `loaded` discovery guard?

Reply:
Not really, since the config flow will abort if there's already an existing config entry, but I'd like to do clean up later separately.